src/components/alert/alert.md

```diff
@@ -1,8 +1,10 @@
-    <div>
```

Comment:
All of these `.md` files are changed because with the new styleguidist, you have to wrap code examples in
\```js
\<Alert />
\```. 

That's the only thing that changed